docs/toc.md

```diff
@@ -7,6 +7,7 @@
 ## [.NET Concepts](standard/concepts.md)
 ## [.NET Standard Library](standard/library.md)
 ## [Frameworks and Targets](standard/frameworks.md)
+## [Choosing between .NET Core and .NET Framework for your server apps](standard/choosing-core-framework-server.md)
```

Comment:
Probably a smaller title for the TOC would work better: 
![image](https://cloud.githubusercontent.com/assets/12971179/20338240/af400416-ab8a-11e6-9843-675d4320c2cb.png)